<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>105</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="btn_edge">
      <property name="styleSheet">
       <string notr="true">font: 63 9pt &quot;Segoe UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Microsoft Edge</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_chrome">
      <property name="styleSheet">
       <string notr="true">font: 63 9pt &quot;Segoe UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Chrome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_yandex">
      <property name="styleSheet">
       <string notr="true">font: 63 9pt &quot;Segoe UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Yandex</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_firefox">
      <property name="styleSheet">
       <string notr="true">font: 63 9pt &quot;Segoe UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>FireFox</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>10</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 10pt &quot;Segoe UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Выбор браузера</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="startup">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>124</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 9pt &quot;Segoe UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Автозапуск</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 10pt &quot;Segoe UI Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
